tests/test_metadata.py

```diff
@@ -207,6 +210,12 @@ def _get_container_label_prefix(
         (pg_container, "postgres", ImageType.APPLICATION)
         for pg_container in POSTGRESQL_CONTAINERS
     ]
+    + [
+        (prom_container, "prometheus", ImageType.APPLICATION)
+        for prom_container in PROMETHEUS_CONTAINERS
+        + ALERTMANAGER_CONTAINERS
+        + BLACKBOX_CONTAINERS
+    ]
```

Comment:
The second value in the parametrization is the container shortname used in the labels:
```suggestion
    + [
        (prom_container, "prometheus", ImageType.APPLICATION)
        for prom_container in PROMETHEUS_CONTAINERS
    ]
    + [
        (alertmngr_ctr, "alertmanager", ImageType.APPLICATION)
        for alertmngr_ctr in ALERTMANAGER_CONTAINERS
    ]
    + [
        (blackbox_ctr, "blackbox_exporter", ImageType.APPLICATION)
        for blackbox_ctr in BLACKBOX_CONTAINERS
    ]
```